build.sbt

```diff
@@ -1612,6 +1612,7 @@ lazy val `runtime-with-polyglot` =
         "ENSO_TEST_DISABLE_IR_CACHE" -> "false"
       ),
       libraryDependencies ++= Seq(
+        "org.graalvm.sdk" % "graal-sdk" % graalVersion % "provided",
```

Comment:
I don't see where it is used in the project

Reply:
`VerifyJavaScriptIsAvailableTest` is using the Graal SDK API. Without the dependency being _"provided"_ the test doesn't compile.